core/src/main/resources/org/apache/spark/ui/static/utils.js

```diff
@@ -46,3 +46,25 @@ function formatBytes(bytes, type) {
     var i = Math.floor(Math.log(bytes) / Math.log(k));
     return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
 }
+
+function padZeroes(num) {
+  return ("0" + num).slice(-2);
+}
+
+function formatTimeMillis(timeMillis) {
+  if (timeMillis <= 0) {
+    return "-";
+  } else {
+    var dt = new Date(timeMillis);
+    return dt.getFullYear() + "-" +
+      padZeroes(dt.getMonth() + 1) + "-" +
+      padZeroes(dt.getDate()) + " " +
+      padZeroes(dt.getHours()) + ":" +
+      padZeroes(dt.getMinutes()) + ":" +
+      padZeroes(dt.getSeconds());
+  }
+}
+
+function getTimeZone() {
+  return new Date().toString().match(/\(([A-Za-z\s].*)\)/)[1];
```

Comment:
how about `Intl.DateTimeFormat().resolvedOptions().timeZone`

Reply:
This timeZone seems incorrect. Safari gets `Asia/Shanghai`, but Chrome gets `America/Chicago` on the same computer.

How about just include this change to release notes.

Reply:
did you have proxy on your chrome? It works fine at my Chrome.